cudamapper/src/main.cpp

```diff
@@ -46,3 +82,18 @@ int main(int argc, char *argv[])
     overlapper.print_paf(overlaps);
     return  0;
 }
+
+void help() {
+    std::cout<<
```

Comment:
Have a look at https://www.youtube.com/watch?v=DiZ-az_nJMM, but this is also fine (and probably even easier to understand).